<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="QWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>532</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="ConnectPLUS">
     <property name="title">
      <string>Connect to PLUS</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Set the Plus Server Host and Name Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="plusHostandPort">
        <property name="text">
         <string>localhost:18944</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="connectPLUSButton">
        <property name="text">
         <string>Connect to PLUS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="ProbeCalibration">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Probe Calibration</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_6">
      <item row="0" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
        <property name="title">
         <string>3D US Image Fiducials</string>
        </property>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0" colspan="2">
          <widget class="qSlicerSimpleMarkupsWidget" name="SimpleMarkupsWidget"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
        <property name="title">
         <string>Probe Fiducials</string>
        </property>
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0" colspan="2">
          <widget class="qSlicerSimpleMarkupsWidget" name="SimpleMarkupsWidget_2"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="USCapture">
     <property name="title">
      <string>Save 3D US Volume</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="Navigation">
     <property name="title">
      <string>Navigation</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Layout:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="layoutComboBox">
        <item>
         <property name="text">
          <string>Conventional View</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red Slice View</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RGBO3D View</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="Settings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save Slicer Scene</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>UpArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save Location:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="savePath">
        <property name="filters">
         <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::Readable|ctkPathLineEdit::Writable</set>
        </property>
        <property name="options">
         <set>ctkPathLineEdit::ShowDirsOnly</set>
        </property>
        <property name="showHistoryButton">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QPushButton" name="testButton">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="NeedleCalibration">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Needle Calibration</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="needlePivotCalibrationButton">
        <property name="text">
         <string>Start Pivot Calibration</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="needleSpinCalibrationButton">
        <property name="text">
         <string>Start Spin Calibration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="needleCalibrationCountdownLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="needleCalibrationErrorLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="PatientRegistration">
     <property name="title">
      <string>Patient Registration</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_11">
      <item row="1" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_3">
        <property name="title">
         <string>In-op Fiducials</string>
        </property>
        <layout class="QFormLayout" name="formLayout_9">
         <item row="0" column="0" colspan="2">
          <widget class="qSlicerSimpleMarkupsWidget" name="SimpleMarkupsWidget_3"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_4">
        <property name="title">
         <string>Pre-op Fiducials</string>
        </property>
        <layout class="QFormLayout" name="formLayout_10">
         <item row="0" column="0" colspan="2">
          <widget class="qSlicerSimpleMarkupsWidget" name="SimpleMarkupsWidget_4"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="CTRegistration">
     <property name="title">
      <string>CT Registration</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="StylusCalibration">
     <property name="title">
      <string>Stylus Calibration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_12">
      <item row="1" column="1">
       <widget class="QPushButton" name="stylusSpinCalibrationButton">
        <property name="text">
         <string>Start Spin Calibration</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="stylusPivotCalibrationButton">
        <property name="text">
         <string>Start Pivot Calibration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="stylusCalibrationCountdownLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="stylusCalibrationErrorLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerSimpleMarkupsWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerSimpleMarkupsWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
